.github/workflows/e2e-conformance-trigger.yaml

```
@@ -1,19 +1,19 @@
 name: E2EConformanceTrigger
 on:
   schedule:
-    - cron: '7 22 * * *'
+    - cron: '7 22 1,15 * *'
```

```suggestion
    - cron: '7 22 * * 1'
```

Can we run this once-a-week during nighttime? I think this is UTC time so can we set the hour to run nightly?